server/models/Board.js

```diff
@@ -0,0 +1,40 @@
+const mongoose = require("mongoose");
+
+const boardSchema = new mongoose.Schema({
+  title: {
+    type: String,
+    required: true,
+    unique: true
+  },
+  creationDate: {
+    type: Date,
+    default: Date.now
+  },
+
+  inProgress: {
+    type: Boolean,
+    default: true
+  },
+
+  isCompleted: {
+    type: Boolean,
+    default: false
+  },
+  
+  owner : {
+    id: {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: 'User'
+    },
+    username: String,
+  },
```

Comment:
This is an odd structure, we can populate the owner just off the ID. If the user changes their username is adds more work to edit all their boards username